src/starknet.rs

```diff
@@ -0,0 +1,157 @@
+mod vec252;
+
+use anyhow::Ok;
+use anyhow::{Context, Result};
+use cairo_felt::Felt252;
+use cairo_lang_runner::{Arg, ProfilingInfoCollectionConfig, RunResultValue, SierraCasmRunner};
+use cairo_lang_sierra::program::VersionedProgram;
+use cairo_lang_utils::ordered_hash_map::OrderedHashMap;
+use cairo_proof_parser::parse;
+use itertools::chain;
+use itertools::Itertools;
+use std::fs;
+use std::io::{BufRead, BufReader};
+use std::path::Path;
+use std::process::{Command, Stdio};
+use vec252::VecFelt252;
+
+use crate::args::ProveArgs;
+
+pub fn run_starknet_verifier(args: &ProveArgs) -> Result<()> {
+    run_locally(&args.output)?;
+
+    Ok(())
+}
+
+#[allow(dead_code)]
+fn run_on_starknet(proof_file: &Path) -> Result<()> {
```

Comment:
I know it's super obvious, but: this code is unused.

Do we still need it or do we wanna get rid of it?

Reply:
Got it rid of it. :)